fetch-nixpkgs.nix

```diff
@@ -20,6 +20,7 @@ let
     inherit input;
     coreutils = builtins.storePath nixcfg.coreutils;
     tar = builtins.storePath nixcfg.tar;
+    shell = builtins.storePath nixcfg.shell;
```

Comment:
@domenkozar do we need the extra dep on `shell` into `master`?

Reply:
Nope.